loop/routes/home.js

```diff
@@ -45,6 +45,16 @@ module.exports = function(app, conf, logError, storage, tokBox) {
             request.put(req.query.SP_LOCATION, function(error, response) {
               pushStatus = (!error && response.statusCode === 200);
               returnStatus(storageStatus, requestError, pushStatus);
+              if (statsdClient !== undefined) {
+                statsdClient.count('loop.simplepush.call', 1);
+                var counter_push_status_counter;
+                if (pushStatus) {
+                  counter_push_status_counter = 'loop.simplepush.call.heartbeat.success';
+                } else {
+                  counter_push_status_counter = 'loop.simplepush.call.heartbeat.failures';
```

Comment:
In production we do care if a Simple Push HTTP request fails with an error. It helps ops detecting what's wrong.